link/src/test/java/com/stripe/android/link/repositories/LinkApiRepositoryTest.kt

```diff
@@ -519,6 +519,39 @@ class LinkApiRepositoryTest {
         )
     }
 
+    @Test
+    fun `deletePaymentDetails without consumerPublishableKey sends correct parameters`() = runTest {
```

Comment:
Maybe a good refactor for a follow up -- it seems like we have to implement the same behavior here for all these API calls and then we also have unit tests for them. Should we just have some shared logic for handling missing publishable keys? And then we can just have a test for that to ensure that works properly

Reply:
Yh, this make sense. I'll look into it for a follow up PR